yt/utilities/answer_testing/framework.py

```diff
@@ -769,9 +773,17 @@ def run(self):
         return result
 
     def compare(self, new_result, old_result):
-        for newp, oldp in zip(new_result["parents"], old_result["parents"]):
+        for newp, oldp in zip(
```

Comment:
this is a good additional check

Reply:
although, it may result in slightly less informative errors if the lengths are different

Reply:
not sure what you mean here: if lengths are different, there might be no error at all (without strict=True)

Reply:
Oh, I just mean, that the old way wouldn't pick that up *as-is* but the new way will show a weird error.  It's NBD.